apps/web/src/app/(profile)/[username]/_components/dashboard/badges/_advent_badges.ts

```diff
@@ -0,0 +1,104 @@
+import type {AllBadgeObjs, BadgeFn} from "../_actions";
```

Comment:
these actions dont belong in the dashboard directory. off the top of my head can probably make a top level actions folder like we do with components.. so `src/actions` since badges can be awarded in many different ways

Reply:
moved to src/actions.